.travis.yml

```diff
@@ -64,6 +79,14 @@ matrix:
       env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
     - php: hhvm-nightly
       env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
+    - php: hhvm-nightly
+      env: DB=sqlanywhere SQLANYWHERE_GDRIVE_ID=$SQLANYWHERE_12_GDRIVE_ID # driver currently unsupported by HHVM
+    - php: hhvm-nightly
```

Comment:
I know but it's currently still present in the `travis.yml` so I adopted it for now. Can be removed completely later with another PR IMO.